docs/source/development/arch_notes.rst

```diff
@@ -0,0 +1,68 @@
+Device Architecture Notes
+=========================
+
+The following notes describe in more details the approach required to add new architectures, or enhance an already existing architecture with additional tiles and primitives.
+
+Adding a new tiles and primitives
+---------------------------------
+
+In Symbiflow Architecture Definitions, this process requires modifications of two parts in the flow which can be summarized in the following points:
+
+- **VTR** Physical Block type definition
+- **Yosys** techmap definition
+
+Physical Block Type Definition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is a VTR-related file which includes a detailed description of the internal logic of the block, as well as its I/O interface.
+
+More details can be gathered in the [official VTR documentation](https://docs.verilogtorouting.org/en/latest/arch/reference/).
```

Comment:
Since this is RST, the syntax for external links is the following:

```
`official VTR documentation <https://docs.verilogtorouting.org/en/latest/arch/reference/>`__.
```

Reply:
Replaced with an interpshinx ref in https://github.com/SymbiFlow/f4pga-arch-defs/pull/1950/commits/a897178cbb053c5924ec9342aa900cd790d3cfa3.